gcloud_bigtable/google/bigtable/admin/cluster/v1/bigtable_cluster_service_api.py

```diff
@@ -0,0 +1,262 @@
+from google.gax import api_utils
+from google.gax import page_descriptor
+from google.longrunning import operations_pb2
+from google.protobuf import timestamp_pb2
+
+class BigtableClusterServiceApi(object):
+  """Service for managing zonal Cloud Bigtable resources."""
+
+  # The default address of the logging service.
+  _SERVICE_ADDRESS = "bigtableclusteradmin.googleapis.com"
+
+  # The default port of the logging service.
+  _DEFAULT_SERVICE_PORT = 443
+
+  # The scopes needed to make gRPC calls to all of the methods defined in
+  # this service
+  _ALL_SCOPES = [
+      'https://www.googleapis.com/auth/bigtable.admin',
+      'https://www.googleapis.com/auth/bigtable.admin.cluster',
+      'https://www.googleapis.com/auth/cloud-bigtable.admin',
+      'https://www.googleapis.com/auth/cloud-bigtable.admin.cluster',
+      'https://www.googleapis.com/auth/cloud-platform'
+  ]
+
+  def __init__(
+      self, service_path=_SERVICE_ADDRESS, port=_DEFAULT_SERVICE_PORT,
+      channel=None, ssl_creds=None, scopes=_ALL_SCOPES,
+      is_idempotent_retrying=True, max_attempts=3, timeout=30):
+
+    self.defaults = api_callable.ApiCallableDefaults(
+      timeout=timeout, max_attempts=max_attempts,
+      is_idempotent_retrying=is_idempotent_retrying)
+
+    self.stub = api_utils.create_stub(
+      bigtable_cluster_service_pb2.beta_create_BigtableClusterService_stub, service_path, port,
+      ssl_creds=ssl_creds, channel=channel, scopes=scopes)
+
+  def __enter__(self):
+    return self
+
+  def __exit__(self, type, value, traceback):
+    self.close()
+
+  def close(self):
+    del self.stub
+
+  # Properties
+  @property
+  def channel(self):
+    return self.channel
+
+  # Page descriptors
+
+  # Service calls
+  def list_zones(self, name="", **kwargs):
+    """Lists the supported zones for the given project."""
+    list_zones_request = bigtable_cluster_service_messages_pb2.ListZonesRequest(
+        name=name, **kwargs)
+    return self.list_zones_callable()(list_zones_request)
+
+  def list_zones_callable(
+      self, is_retrying=None, max_attempts=None):
+    return api_callable.idempotent_callable(
+        self.stub.ListZones,
+        is_retrying=is_retrying,
+        max_attempts=max_attempts,
+        defaults=self.defaults)
+
+  def get_cluster(self, name="", **kwargs):
+    """Gets information about a particular cluster."""
+    get_cluster_request = bigtable_cluster_service_messages_pb2.GetClusterRequest(
+        name=name, **kwargs)
+    return self.get_cluster_callable()(get_cluster_request)
+
+  def get_cluster_callable(
+      self, is_retrying=None, max_attempts=None):
+    return api_callable.idempotent_callable(
+        self.stub.GetCluster,
+        is_retrying=is_retrying,
+        max_attempts=max_attempts,
+        defaults=self.defaults)
+
+  def list_clusters(self, name="", **kwargs):
+    """
+    Lists all clusters in the given project, along with any zones for which
+    cluster information could not be retrieved.
```

Comment:
We are trying to re-use the language-independent proto doc. We can address some of these with changes to the guidelines of those docs. So requiring that the first sentence of doc is a concise description is possible, but we would want to understand the consequences of breaking PEP257 since, changes to these doc standards are costly to enforce.